### PR TITLE
API Center SDK Update

### PR DESCRIPTION
**Pull request by alexyaang**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/apicenter/Azure.ResourceManager.ApiCenter/CHANGELOG.md
Comment (ArthurMa1978) [inline sdk/apicenter/Azure.ResourceManager.ApiCenter/CHANGELOG.md:3]: Can this be stable as the swager spec is stable version.

**Review by jsquire — CHANGES_REQUESTED** [review]
@alexyaang :

You are missing the needed permissions and public group memberships for an internal contributor.  Please review the Azure SDK onboarding documentation and use the onboarding teams channel for any questions.
- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

File: sdk/apicenter/Azure.ResourceManager.ApiCenter/CHANGELOG.md
Comment (alexyaang) [reply sdk/apicenter/Azure.ResourceManager.ApiCenter/CHANGELOG.md:3]: @azaslonov thoughts?

**Comment by alexyaang** [comment]
@jsquire my apologies for misreading the onboarding guide. I have adjusted my `@Azure` org membership visibility, requested for permission to join the partner group (still pending approval), and added me and @azaslonov to the owners file as owners of `sdk/apicenter`. That seems to be all from the documentation, but please let me know if I'm missing something.

File: sdk/apicenter/Azure.ResourceManager.ApiCenter/CHANGELOG.md
Comment (azaslonov) [reply sdk/apicenter/Azure.ResourceManager.ApiCenter/CHANGELOG.md:3]: Yes please, it should be stable 1.0.0 version. @alexyaang could you please change?

Comment (alexyaang) [reply sdk/apicenter/Azure.ResourceManager.ApiCenter/CHANGELOG.md]: changed it to `1.0.0`, good catch guys

**Comment by alexyaang** [comment]
@jsquire Update: I have joined the partner group & adjusted my @Azure org membership visibility. 

I requested review from you

File: .github/CODEOWNERS
Comment (jsquire) [inline .github/CODEOWNERS:232]: This needs to move down to the management section, around L930.   Please organize to preserve alphabetical ordering.

Comment (jsquire) [inline .github/CODEOWNERS:232]:
```suggestion
/sdk/apicenter/Azure.ResourceManager.ApiCenter/                     @alexyaang @azaslonov
```

File: .github/CODEOWNERS
Comment (jsquire) [inline .github/CODEOWNERS:234]:
```suggestion
# ServiceLabel: %API Center %Mgmt
```

**Review by jsquire — DISMISSED** [review]
@alexyaang:

Your account still lacks the needed write permissions of a Microsoft contributor/Code owner.

File: .github/CODEOWNERS
Comment (alexyaang) [reply .github/CODEOWNERS]: I think I get what you mean? I made a commit here https://github.com/Azure/azure-sdk-for-net/pull/43013/commits/af80885324bea1030fecb2d186f2aded9bd247f5, is this what you meant?

**Comment by alexyaang** [comment]
@ArthurMa1978 I believe all this PR needs is management approval from you, just an FYI. Thanks in advance

**Comment by alexyaang** [comment]
> @alexyaang: Please do not ever remove the review of an Azure SDK team member without their explicit consent.

Sorry, not sure what you're referring to, what did I do exactly?

**Comment by alexyaang** [comment]
@jsquire Ah okay, sorry didn't mean to do that. I thought it was equivalent of resolving a comment

File: sdk/apicenter/Azure.ResourceManager.ApiCenter/assets.json
Comment (ArthurMa1978) [inline sdk/apicenter/Azure.ResourceManager.ApiCenter/assets.json:4]: Since this is stable release please follow
[azure-sdk-for-net/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md at main · Azure/azure-sdk-for-net (github.com)](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md)
to add tests and record the test result

**Pull request closed without merging**
